addons/portal/static/src/scss/portal.scss

```diff
@@ -288,7 +288,7 @@ img, .media_iframe_video, .o_image {
 // ===  Page custom bg ===
 // To be applied to all portal pages if bg-color is white (default).
 @if ($o-portal-use-default-colors) {
-    #wrapwrap.o_portal {
+    #wrapwrap.o_portal > main {
```

Comment:
That won't work in boxed layout, you will still see the default white around header and footer. I can't think of a good solution, the correct one is extending `$o-color-extras-nesting-selector` with `#wrapwrap.o_portal` but this would create much CSS for just this small usecase :/

A better solution would be to have SCSS dedicated to the portal pages instead of a wrapping selector loaded by everyone but this is a master solution :/

Maybe we can compromise for stable: using the o-bg-color mixin with `$with-extras: false` for this usecase here... that would break text muted in portal for apps which customizes `$o-portal-bg-color` to a dark bg but this is probably not the case anywhere and I can't think of better (and as soon as the user customize the body bg color, the bug is gone anyway, this bug only exists for non-customized default palette color).

Reply:
Ok, I am changing to using the mixin with ´$with-extras: false´